astropy/units/structured.py

```diff
@@ -3,7 +3,7 @@
 This module defines structured units and quantities.
 """
 
-from __future__ import annotations  # For python < 3.10
+from __future__ import annotations
```

Comment:
If this was for `python < 3.10`, can we not just remove this import?

Reply:
I wasn't sure at first, since this import has been propagating in the code base for seemingly different reasons since this comment was written, but upon checking I think you're right. Thanks !

Reply:
For here probably fine, but let me ping @nstarman to ask if we still need `from __future__ import annotations` in general now that our minimum python is 3.10.